app-backend/db/src/main/scala/ObjectPermissions.scala

```diff
@@ -92,6 +92,7 @@ trait ObjectPermissions[Model] {
   def getPermissions(
       id: UUID): ConnectionIO[List[Option[ObjectAccessControlRule]]] =
     for {
+      // TODO restrict to the user's permissions if the user does not have edit permissions
```

Comment:
Currently only users with edit permissions have permission to hit this endpoint - frontend will assume that they do not have edit permissions if the endpoint returns a 403 and handles accordingly